### PR TITLE
fix/docker compose

### PR DESCRIPTION
**Pull request by anwalker293**

### CODE REVIEW

**Comment by anwalker293** [comment]
I think this is ready to go, but I'll wait until I finish making all of the changes before calling this more attention

**Comment by anwalker293** [comment]
(Need to change url, will do so when I have the next free change)

**Comment by anwalker293** [comment]
I think this is ready to go, but I think there may be weird spacing around alice's --genesis-url

File: docker-compose.yml [hunk 1]
```diff
@@ -18,23 +26,28 @@ services:
         --admin 0.0.0.0 3001 --admin-insecure-mode
         --log-level debug
         --webhook-url http://echo:3002/webhook
-        --genesis-url https://raw.githubusercontent.com/Indicio-tech/indicio-network/main/genesis_files/pool_transactions_demonet_genesis
-        --wallet-type askar
+      	--genesis-url https://raw.githubusercontent.com/Indicio-tech/indicio-network/main/genesis_files/pool_transactions_testnet_genesis
```
Comment (dbluhm) [inline docker-compose.yml:29]: Sorry, nitpick: whitespace issues here. Likely a tab where spaces were intended

**Comment by anwalker293** [comment]
@dbluhm Fixed! :)

**Pull request merged**